### ISSUE TITLE
transient incorrect missing value with @SHF

### ISSUE THREAD

**Issue by karlmsmith** [NOAA-PMEL/Ferret#1586]
**Reported by andrew wittenberg on 11 Sep 2015 22:19 UTC**
The following returns an incorrect missing value:

```
 	NOAA/PMEL TMAP
 	FERRET v6.95 (rc2)
 	Linux 2.6.32-504.23.4.el6.x86_64 64-bit - 06/17/15
 	11-Sep-15 18:13     

yes? use monthly_navy_winds; set reg/x=180/y=0
yes? list/nohead/l=131 uwnd[       ....
```

The value should in fact be valid, since UWND has 132 L-points.

This bug is somehow transient, since if we next do

```
yes? list/nohead/l=130:131 uwnd[l=@shf](l=@shf]
)
 17-OCT-1992 / 130: -0.7831
 16-NOV-1992 / 131: -0.2641
```

then we now get back a correct value for the very same command that failed above:

```
yes? list/nohead/l=131 uwnd[l=@shf]
         -0.2641
```

Migrated-From: http://dunkel.pmel.noaa.gov/trac/ferret/ticket/2314

**Comment by karlmsmith** [comment]
**Comment by andrew.wittenberg on 11 Sep 2015 22:19 UTC**
Could this be related to #1462 or #2301 ?

**Comment by karlmsmith** [comment]
**Comment by ansley.b.manke on 12 Sep 2015 00:20 UTC**
Whatever it is, it's recent; the behavior doesn't show up in v6.93.

**Comment by karlmsmith** [comment]
**Comment by ansley.b.manke on 15 Sep 2015 18:14 UTC**
This is fixed. It was caused by the fix for #2301 which I've worked on further. (The other related bugs that were also corrected when #2301 was fixed are still fixed; #1462, #2300)
